src/MongoDB/Driver/ReadPreference.cpp

```diff
@@ -152,8 +152,8 @@ Array HHVM_METHOD(MongoDBDriverReadPreference, __debugInfo)
 		retval.set(s_tags, v_tags.toArray());
 	}
 
-	if (mongoc_read_prefs_get_max_staleness_ms(data->m_read_preference) != 0) {
-		retval.set(s_maxStalenessMS, mongoc_read_prefs_get_max_staleness_ms(data->m_read_preference));
+	if (mongoc_read_prefs_get_max_staleness_seconds(data->m_read_preference) != -1) {
```

Comment:
I believe `NO_MAX_STALENESS` will end up being a public constant, so you may want to leave a TODO to update this later (see: [CDRIVER-1925](https://jira.mongodb.org/browse/CDRIVER-1925)). That goes for other places where I talked about detecting -1 as well.